index.html

```diff
@@ -337,13 +338,14 @@ <h1>Get Started</h1>
                     <div class="email">
                         <label for="email">E-mail</label>
                         <br>               
-                        <input type="text" id="email" name="email" placeholder="Your e-mail..">
+                        <input type="text" id="email" name="_replyto" placeholder="Your e-mail..">
```

Comment:
- [ ] The `email` input should have an `email` type attribute and `required` attribute to activate the validation on the form submit. Kindly fix it.

Reply:
Changed. Thanks for this review.